examples/mixPureFusionMaterials.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 import material_db_tools as mdbt
```

Comment:
Should we change it so it can pull directly from the file instead of us having to add the file to the same folder with the createPure script? Or is that just a me problem?

`sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))`
Or is that too much?

Reply:
I just have the directories on my PYTHONPATH. I think there are better ways of doing this that we could look at going forward

Reply:
I think PYTHONPATH is probably the best for if you're using it regularly. But might not be the best for people that just want to run the scripts. But I agree I have no clue what best practice is.

Reply:
Ultimately, we'll get to a place where we install it correctly with a `pyproject.toml` etc... so I think just managing this manually for the moment is OK